api/src/main/java/com/okta/authn/sdk/resource/HardwareFactorProfile.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020-Present Okta, Inc.
```

Comment:
Another easy way to find the original date is to search for the class name here:
https://oss.sonatype.org/#nexus-search;classname~com.okta.sdk.resource.user.factor.HardwareFactorProfile